changelog.md

```diff
@@ -38,37 +46,6 @@
 - Microsoft.Identity.Web nows processes the `AcceptHeader` and `ContentType` if provided, see issue [#2806](https://github.com/AzureAD/microsoft-identity-web/issues/2806) for details.
 - Target Microsoft.IdentityModel 7x in OWIN targets, see issue [#2785](https://github.com/AzureAD/microsoft-identity-web/issues/2785) for details. 
 
-========
-
-**See [rel/v2 changelog](https://github.com/AzureAD/microsoft-identity-web/blob/rel/v2/changelog.md#2200) for full list of added features to 3.0.0**.
-  
-========
-
-2.19.0
-=========
-- Updated to Microsoft.IdentityModel.* 7.6.0
-
-### New features
-- Id Web now provides a `.WithUser()` modifier to the Microsoft Graph queries (like `WithAppOnly()`). See issue [#2855](https://github.com/AzureAD/microsoft-identity-web/issues/2855) for details.
-- Id Web now provides a base class for implementing a custom `IAuthorizationHeaderProvider`. See issue [#2856](https://github.com/AzureAD/microsoft-identity-web/issues/2856) for details.
-
-### Bug Fixes
-- Id Web now processes the extra query parameters when included as part of the authority. See issue [#2697](https://github.com/AzureAD/microsoft-identity-web/issues/2697) for details.
-- If `Logging:LogLevel:Microsoft.Identity.Web` is assigned to `None`, no default logger is initialized and Microsoft.Identity.Web does not record any logs. See [#2816](https://github.com/AzureAD/microsoft-identity-web/pull/2816) for details. 
```

Comment:
I don't see changes from line 57-59 as part of the changelog in here https://github.com/AzureAD/microsoft-identity-web/blob/rel/v2/changelog.md#2200 ?

Reply:
Good catch, as far as I can tell only PR 2840 ever went into rel/v2 so I  propose adding that one to rel/v2 in [this other PR](https://github.com/AzureAD/microsoft-identity-web/pull/2970/files) .

For the other two lines I'll add them to the 3.0.0-preview2 section which is the next release after they were merged.